src/containers/pages/IndexPage/index.js

```diff
@@ -50,9 +50,10 @@ const IndexPage = ({ classes, onClickGoToRepositories }) => (
         <Typography align='justify' className={ classes.gridDescription }>
           Easily setup a benchee/config.yml and the benchmark scripts to start collecting
           performance metrics for your project. We currently support benchmarks
-          written with Benchee, a powerful library specially designed to test
-          and measure performance of Elixir code. See the examples of the
-          Ecto library.
+          written with <a href='https://github.com/PragTob/benchee'><u>Benchee</u></a>,
+          a powerful library specially designed to test
+          and measure performance of Elixir code. Visit some benchmarks examples of the
+          <a href='https://github.com/elixir-ecto/ecto'> <u>Ecto library</u></a>.
```

Comment:
Usage of `<u>` is deprecated, we should either style links differently through css or if we want to set an emphasis do `<em>` or something

Reply:
changed!